docs/samples/Microsoft.ML.Samples.GPU/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Reflection;
+using Samples.Dynamic;
+
+namespace Microsoft.ML.Samples
+{
+    public static class Program
+    {
+        public static void Main(string[] args) => RunAll();
+
+        internal static void RunAll()
```

Comment:
Why is this empty?

Reply:
As I wrote in [the issue](https://github.com/dotnet/machinelearning/issues/4299#issuecomment-538587038), I got runtime errors if I tried to run it with the original content of Program.cs. So I removed that.

Should I put that back into the method?

Reply:
What were the errors? Did you run the samples on a machine with a GPU?
If there are errors, then the samples that throw them should be fixed.

---
In reply to: [331726596](https://github.com/dotnet/machinelearning/pull/4300#discussion_r331726596) [](ancestors = 331726596)

Reply:
My machine doesn't have a dedicated GPU. And now I realize that the sample didn't run because I don't have tensorflow installed either.

I will find a machine that has those to run the samples, and if it works then I will return the content of Program.cs.

Thanks for the feedback.

Reply:
So I had checked this on another machine, and the samples run with the original content of the Program.cs file, so I've put that back in here.

So, please, approve this PR. Thanks!

Reply:
So today I got my machine with a GPU running. And I checked that this works as expected; everything went correctly. Please, approve this PR. Thanks!

Reply:
Thanks for verifying that the samples run correctly!
Since they are, is this PR still needed? This file is identical to the one in Microsoft.ML.Samples, isn't it?

---
In reply to: [337241467](https://github.com/dotnet/machinelearning/pull/4300#discussion_r337241467) [](ancestors = 337241467)

Reply:
Yes, they are identical. The problem was that both projects shared the same file, and when editing the other file to run a specific sample, then some error tooltips appeared when editing it or when trying to rebuild the solution after making those changes, because ML.Samples.GPU doesn't have access to the same things that ML.Samples.

So these errors didn't appear before, and after talking about it with @codemzs we agreed that I should add another Program.cs file in ML.Samples.GPU with the same content of the one in ML.Samples.